builder/vsphere/iso/step_create.go

```diff
@@ -64,52 +67,61 @@ type NIC struct {
 	// - Logical Switch UUID: `<uuid>`
 	// - Segment ID: `/infra/segments/<SegmentID>`
 	//
-	// ~> **Note:** If more than one network resolves to the same name, either the inventory path to
-	// network or an ID must be provided.
+	// ~> **Note:** If more than one network resolves to the same name, either
+	// the inventory path to network or an ID must be provided.
 	//
-	// ~> **Note:** If no network is specified, provide `host` to allow the plugin to search for an
-	// available network.
+	// ~> **Note:** If no network is specified, provide `host` to allow the
+	// plugin to search for an available network.
 	Network string `mapstructure:"network"`
-	// Specifies the virtual machine network card type. For example `vmxnet3`.
+	// The virtual machine network card type. For example `vmxnet3`.
 	NetworkCard string `mapstructure:"network_card" required:"true"`
-	// Specifies the network card MAC address. For example `00:50:56:00:00:00`.
+	// The network card MAC address. For example `00:50:56:00:00:00`.
 	MacAddress string `mapstructure:"mac_address"`
-	// Specifies whether to enable DirectPath I/O passthrough for the network device.
+	// Enable DirectPath I/O passthrough is for the network device.
```

Comment:
```suggestion
	// Enable DirectPath I/O passthrough for the network device.
```

^ Is this what you were aiming for? Otherwise I've probably misunderstood what the docs were saying, please feel free to correct me as you deem necessary

Reply:
Addressed in ccab453.